Docker/README.md

```diff
@@ -0,0 +1,103 @@
+Docker containers by themselves are stateless, so in order for the container to act as a stateful container, some local preparations and additional runtime arguments are needed to be passed into the Docker run command.
+
+## Setting up O3DE-specific folders
+O3DE will use two folders for its runtime environment. The manifest folder (\$HOME/.o3de) and the O3DE Home folder (\$HOME/O3DE). On normal installations/build of the full O3DE engine, these folders are created on-demand based on the current logged in user. Since Docker containers by default are stateless, all the information that is downloaded, updated, deleted, etc from these two folders as part of normal operations will no longer exist once the container is disconnected, thus losing any work that was done while using O3DE. In order to utilize the O3DE Docker container to create and maintain O3DE projects beyond the container session, we will need to create these two folders manually and pass them in as mapped volumes to its corresponding specific folders within the container. 
+
+The following example creates these folders under a parent `docker` folder in the current user's `$HOME` directory, specific to the name of the Docker image.
+
+```
+mkdir -p $HOME/docker/o3de/manifest
+
+mkdir -p $HOME/docker/o3de/home
+```
+
+>**Note** The two folders must be created specifically for the O3DE Docker container that is being connected to. There is no version or compatibility checks in the Docker container, so normal O3DE startup in the container may fail if it does not use the same mapped folders it used during its initial launch.
+
+These two folders will be added as part of the `docker run` argument list:
+
+```
+-v "$HOME/docker/o3de/manifest:/home/o3de/.o3de" -v "$HOME/docker/o3de/home:/home/o3de/O3DE"
+```
+
+## Using the current user id and group id
+The Docker image will map the current user and group id to a Docker create user and group when running the container. This will allow reading and writing to the mapped O3DE-specific folders described earlier and will add persistence to the Docker container through the mapped folders. The user id and group id are passed to the Docker run command as two arguments: `UID` and `GID`:
+
+```
+--env UID=$(id -u) --env GID=$(id -g)
+```
+
+
+Putting it all together, you can launch the O3DE Docker container with the following command:
+
+```
+xhost +local:root
+
+docker run --env UID=$(id -u) --env GID=$(id -g) -v "$HOME/docker/o3de/manifest:/home/o3de/.o3de" -v "$HOME/docker/o3de/home:/home/o3de/O3DE" --rm --gpus all -e DISPLAY=:1 -v /tmp/.X11-unix:/tmp/.X11-unix -it amd64/o3de:ubuntu.jammy
+```
+
+For the ROS container based on the **ROS Humble** example, add the `ros.humble` tag.
+```
+xhost +local:root
+
+docker run --env UID=$(id -u) --env GID=$(id -g) -v "$HOME/docker/o3de/manifest:/home/o3de/.o3de" -v "$HOME/docker/o3de/home:/home/o3de/O3DE" --rm --gpus all -e DISPLAY=:1 -v /tmp/.X11-unix:/tmp/.X11-unix -it amd64/o3de:ros.humble
+```
```

Comment:
I wonder if it would make sense to add one or two lines describing what the outcome could look like after running these commands.